demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,150 @@
+# Python* Model API package
+
+Model API package is a set of wrapper classes for particular tasks and model architectures, simplifying data preprocess and postprocess as well as routine procedures (model loading, asynchronous execution, etc...)
+Model API wrappers hide the specific code inside and work as black-box: application feeds model class with input data, then the model returns post-processed output data in user-friendly format.
+
+## Package structure
+
+The Python* Model API consists of 3 libraries:
+* _adapters_ - implement a common interface to allow Model API wrappers usage with different executors: OpenVINO, ONNX, etc. See [Model API adapters](#model-api-adapters) section
+* _models_ - implement wrappers for each architecture. See [Model API Wrappers](#model-api-wrappers) section
+* _pipelines_ - implement pipelines for model inference and manage the synchronous/asynchronous execution. See [Model API Pipelines](#model-api-pipelines) section
+
+### Prerequisites
+
+The package requires
+- Python (version 3.6 or higher)
+- OpenVINO™ toolkit
+
+If you build Python* Model API package from source, you should install the OpenVINO™ toolkit. See the options:
+
+Use installation package for [Intel® Distribution of OpenVINO™ toolkit](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit-download.html) or build the open-source version available in the [OpenVINO GitHub repository](https://github.com/openvinotoolkit/openvino) using the [build instructions](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode).
+
+Also, you can install the OpenVINO Python\* package via the command:
+ ```sh
+pip install openvino
+ ```
+
+## Building Python* Model API package
+
+Use the following command to install Python* Model API from source:
+```sh
+pip install <omz_dir>/demos/common/python
+```
+
+Alternatively, you can generate the package using a wheel. Follow the steps below:
+1. Build the wheel.
+
+```sh
+python <omz_dir>/demos/common/python/setup.py bdist_wheel
+```
+The wheel should appear in the dist folder.
+Name example: `openmodelzoo_modelapi-0.0.0-py3-none-any.whl`
+
+2. Install the package in the clean environment with `--force-reinstall` key.
+```sh
+pip install openmodelzoo_modelapi-0.0.0-py3-none-any.whl --force-reinstall
+```
+
+To verify the package is installed, you might use the following command:
+```sh
+python -c "from openvino.model_zoo import model_api"
+```
+
+## Model API Wrappers
+
+The Python* Model API package suggests ready-to-use model wrappers, which implement standardized preprocessing/postprocessing functions per "task type" and incapsulate model-specific logic for usage of different models in a unified manner inside the application.
+
+The wrapper interface is simple and flexible, which gives capabilities for the creation of custom wrappers covering different architectures and use cases.
+
```

Comment:
```suggestion
```
It doesn't bring anything

Reply:
Why? Shouldn't we mention that one might create custom wrapper class according to the flexible interface? @vladimir-dudnik

Reply:
That's Python. Anyone can do anything with your classes

Reply:
Did not get your point. Anyone can do anything, including "creation of custom wrappers covering different architectures and use cases." Then statement is not wrong?

Reply:
The statement is not wrong, but
> It doesn't bring anything

Reply:
ok, let's remove this statement